tests/test_schema_verifier.py

```
@@ -577,7 +670,451 @@ def test_complex_nested_structure(self, verifier):
+            "schema = {'type': 'object', 'properties': {'name': {'type': 'string'}}}\n"
+            "r = SchemaVerifier().verify({'name': 'John'}, schema)\n"
+            "print(r.proof_ref)\n"
+        )
+        env = dict(os.environ)
+        root = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+        src = os.path.join(root, "src")
+        env["PYTHONPATH"] = src + os.pathsep + env.get("PYTHONPATH", "")
+
+        outputs = []
+        for _ in range(2):
+            proc = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)